File: app/src/main/java/com/softaai/designpatterns/java/creational/builder/meal/item/Burger/VegBurger.java
package com.softaai.designpatterns.java.creational.builder.meal.item.Burger;

/**
 * Created by Amol Pawar on 01-04-2019.
 * softAai Apps
 */
public class VegBurger extends Burger {
    @Override
    public String name() {
        return "Veg Burger";
    }

    @Override
    public float price() {
        return 25.0f;
    }
}

File: app/src/main/java/com/softaai/designpatterns/kotlin/creational/builder/meal/item/Burger/VegBurger.kt
package com.softaai.designpatterns.kotlin.creational.builder.meal.item.Burger


/**
 * Created by Amol Pawar on 01-04-2019.
 * softAai Apps
 */
class VegBurger : Burger() {
    override fun price(): Float {
        return 25.0f
    }

    override fun name(): String {
        return "Veg Burger"
    }
}